### DISCUSSION TITLE
Looking for release notes?

### DISCUSSION THREAD

**Discussion by OranguTech** [d365collaborative/d365fo.tools#781] [General]
I run update-module, see a new version installs, there's nothing in https://github.com/d365collaborative/d365fo.tools/releases, nothing in the root readme I can find. Am I just blind/overlooking something? 🤷

**Comment by FH-Inway** [comment]
You are not blind, our release notes are pretty much nonexistent. @Splaxi usually makes a post on X for the bigger releases and highlights the most exciting changes. But apart from that, I guess the [commit log](https://github.com/d365collaborative/d365fo.tools/commits/development/)?

@Splaxi Do you think adding GitHub releases could be worthwhile? GitHub has some nice automation that generates the release notes (it lists the pull requests, generates a link to the commit log and lists the contributors - see https://github.com/microsoft/2LCS/releases/tag/v.0.44.0 for an example of how this would look like). Apart from providing release notes, people could also use the GitHub release watch feature to get notified in case of new releases.
![image](https://github.com/d365collaborative/d365fo.tools/assets/33372796/a513bb2f-f58f-40dd-9c8a-43ff6237f356)

**Comment by Splaxi** [comment]
Ah - yes, release notes.

I'm up for any thing that makes a difference and provides value, as long as it doesn't increase the over all effort that we have to put into making a release.

The tools are high pace, when we have breaking changes or when we hit new stuff, that needs to have some automation done. This is top priority - with that in mind, let's make stuff happen 🤘

**Reply by OranguTech** [reply]
Thanks - I'm great at pointing out issues, not so great at fixing them 😈, but let me if there's some way for outsiders to help on this part.  IIRC there's really only a couple people contributing, but automated release notes (even if just a list of included PRs/Bugs) goes a long way.

**Comment by FH-Inway** [comment]
@OranguTech https://github.com/d365collaborative/d365fo.tools/releases/tag/0.7.8 :)
Thanks @Splaxi
